tests/error_016_import_map_malformed.test

```diff
@@ -0,0 +1,4 @@
+args: run --reload --importmap=tests/importmaps/malformed_import_map.json_ tests/importmaps/test.ts
```

Comment:
integration tests are relatively close compared to unit tests. Is it possible to test this within cli/importmap.rs ?

Reply:
@ry I would only be able to test if `ImportMap::from_json()` and friends return a nice `ImportMapError` instead of panicking, which they already did. The problem was that this `ImportMapError` was being unwrapped in main.rs.